src/main/resources/templates/widget.html

```diff
@@ -0,0 +1,98 @@
+<div
+  layout:decorate="~{layout/base}"
+  layout:fragment="content"
+  xmlns:th="http://www.thymeleaf.org"
+  xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+>
+  <div class="container" th:classappend="${additionalClass}">
+    <div class="widget">
+      <div
+        th:if="${localizationError}"
+        th:insert="fragments/error :: notification"
+      ></div>
+      <div class="cards">
+        <img
+          class="img-responsive"
+          src="images/id-1x.png"
+          th:alt="${localization.headlineImageAlt}"
+          srcset="images/id-2x.png 2x"
+        />
+        <img
+          class="app-icon"
+          src="images/app-icon.svg"
+          th:alt="${localization.appIconImageAlt}"
+        />
+      </div>
+      <div class="headline">
+        <h1>
+          [[${localization.headlineTitleTop}]] <br />
+          [[${localization.headlineTitleBottom}]]
+        </h1>
```

Comment:
I just realise, as you mentioned in my commit merge, `h1` should be available once in a page, since our iframe means embedded in parent page, then we should not use `h1` right because the parent page (GST) has `h1` in place already